arch/xtensa/Kconfig

```diff
@@ -128,9 +128,17 @@ config XTENSA_USE_SEPARATE_IMEM
 		This separate heap will be part of the internal DRAM.  It starts right after .data
 		and ends at the configurable size given by the OPTION XTENSA_IMEM_REGION_SIZE.
 
+config XTENSA_IMEM_REGION_SIZE_MAX
```

Comment:
Although `XTENSA_IMEM_REGION_SIZE_MAX` and `XTENSA_IMEM_REGION_SIZE` share similar names, they refer to different aspects of the memory.
`XTENSA_IMEM_REGION_SIZE` refers to the **DRAM** region size, while `XTENSA_IMEM_REGION_SIZE_MAX` refers just to the heap region.
Furthermore, I suggest to rename `XTENSA_IMEM_REGION_SIZE_MAX` to `XTENSA_IMEM_MAXIMIZE_HEAP_REGION`, so it becomes more adequate to a **bool** config, which intends to specify an action.

Reply:
Agree, thanks for your comments, I have corrected it.